.gitignore

```diff
@@ -37,3 +37,6 @@ crates/python/tests/__output__/
 
 # unversioned developer notes
 scratch/
+
+# un-consolidated updates to THIRDPARTY license data
+/THIRDPARTY-update.yaml
```

Comment:
This is to facilitate re-running `cargo bundle-licenses` and comparing the output with the existing output, even though there's not yet any tooling to assist with doing so.